src/kkt.c

```diff
@@ -34,7 +34,7 @@ static void KKT_dealloc(KKTObject *self) {
 KKTObject *KKT_New(void) {
   KKTObject *k = (KKTObject*)PyObject_New(KKTObject, &KKTType);
   if (k==NULL) return k;
-  bzero((void*)(&(k->kkt)), sizeof(pyglpk_kkt_t));
+  memset((void*)(&(k->kkt)), 0, sizeof(pyglpk_kkt_t));
```

Comment:
Small nitpick: The cast to `void *` is not [needed][0] and because of operator precedence the parentheses around `k->kkt` are [unneeded][1]. I think

```c
memset(&k->kkt, 0, ...
```

is a bit easier to parse.

[0]: http://stackoverflow.com/a/16534673
[1]: https://en.wikipedia.org/wiki/Operators_in_C_and_C%2B%2B#Operator_precedence